user-interface/src/login/providers/okta/okta-library.ts

```diff
@@ -42,12 +42,13 @@ async function refreshTheToken(oktaAuth: OktaAuth) {
     const oktaUser = await oktaAuth.getUser();
     if (accessToken) {
       const jwt = oktaAuth.token.decode(accessToken);
+      // TODO: THIS REFRESH IS NOT "AUGMENTED". WE NEED TO CALL THE /me ENDPOINT.
```

Comment:
Can we just lean on Session.tsx to write the session to local storage since it is integrated with the /me endpoint?